UnitedIncome/components
# Changes publishing pipeline

### Description
> Please provide a brief description of your changes below.

Changes the publishing pipeline so `develop` goes into `/develop`, and `master` sits int he root.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

I guess we just run it and find out

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
